programs/drift/src/state/state.rs

```diff
@@ -54,6 +54,7 @@ pub enum ExchangeStatus {
     LiqPaused = 0b00010000,
     FundingPaused = 0b00100000,
     SettlePnlPaused = 0b01000000,
+    SkipAuctionDurationPaused = 0b10000000,
```

Comment:
nit: SkipAmmImmediateFill might be more obvious name

Reply:
made it `AmmImmediateFillPaused` to follow pattern